php/libraries/NDB_Client.class.inc

```diff
@@ -104,7 +112,8 @@ class NDB_Client
             . "default-src 'self' 'unsafe-inline'; "
             . "script-src 'self' 'unsafe-inline' 'unsafe-eval'; "
             . "font-src 'self' data:; "
-            . "img-src 'self' data:"
+            . "img-src 'self' data:; "
+            . $config_additions
```

Comment:
1. i tested that, no effect seems to work fine
2. i did not try all combinations but it should be the same as wt would happen if you entered them directly in the code... which is what is being done right now